.github/ISSUE_TEMPLATE/Bug_issue.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Report a bug
+about: Report an issue that you have encountered.
+title: ''
+labels: 'bug'
+---
+#### Summary
+
+<!--Please describe concisely the error you are encountering, specifying what goes wrong and what should have happened. If you get an error message, please post it completely. Consider also to post an image to illustrate more clearly if appropriate.-->
+
+#### Additional information
+
+<!--Provide CIL version and OS version-->
+<!--e.g. import cil; print(cil.version)-->
```

Comment:
I would remove the comment and add:

Attach the output of 
```
python -c "import cil, sys; print(cil.version.commmit_hash, sys.version, sys.platform)"

```

Reply:
```suggestion
<!--Version info: please post the output of the following command-->
`python3 -c "import cil, sys; print(cil.version.commmit_hash, sys.version, sys.platform)"`
```